xbmc/network/Socket.cpp

```diff
@@ -69,7 +69,7 @@ bool CPosixUDPSocket::Bind(bool localOnly, int port, int range)
           for (m_iPort = port; m_iPort <= port + range; ++m_iPort)
           {
             m_addr.saddr.saddr6.sin6_port = htons(m_iPort);
-            if (bind(testSocket, (struct sockaddr*)&m_addr.saddr, m_addr.size) >= 0)
+            if (bind(testSocket, static_cast<struct sockaddr*>(&m_addr.saddr), m_addr.size) >= 0)
```

Comment:
Sorry, has testet for my comment now. Should be `reinterpret_cast` otherwise it brings compile fault.